test/built-ins/Temporal/PlainDateTime/hour-undefined.js

```diff
@@ -5,12 +5,19 @@
 esid: sec-temporal.plaindatetime
 description: Hour argument defaults to 0 if not given
 features: [Temporal]
+includes: [temporalHelpers.js]
 ---*/
 
 const args = [2000, 5, 2];
 
 const explicit = new Temporal.PlainDateTime(...args, undefined);
-assert.sameValue(explicit.hour, 0, "hour default argument");
+
+TemporalHelpers.assertPlainDateTime(
+  explicit,
+  2000, 5, "M05", 2, 0, 0, 0, 0, 0, 0,
+  "hour default argument (argument present)"
+);
 
 const implicit = new Temporal.PlainDateTime(...args);
-assert.sameValue(implicit.hour, 0, "hour default argument");
+
+TemporalHelpers.assertPlainDateTimesEqual(explicit, implicit, "hour default argument (argument missing)");
```

Comment:
I'd prefer just using `TemporalHelpers.assertPlainDateTime` here as well; I don't think comparing several objects under test is as useful as testing the expectations directly.

Reply:
done